dolphinscheduler-ui-next/package.json

```diff
@@ -17,7 +17,7 @@
     "echarts": "^5.2.2",
     "lodash": "^4.17.21",
     "monaco-editor": "^0.31.1",
-    "naive-ui": "2.23.2",
+    "naive-ui": "2.24.6",
```

Comment:
Newly introduced dependencies?

Reply:
No, it's a dependency upgrade

Reply:
Looks like a version update. You may need to check the corresponding license

Reply:
license has not changed

Reply:
After the new ui development is basically completed, I will deal with the license

Reply:
👍 Great, thanks @CalvinKirs point it out